fluent-package/yum/fluent-package.spec.in

```diff
@@ -188,8 +188,12 @@ else
     fi
 fi
 if [ $1 -eq 2 ]; then
-  # collect installed gems during upgrading
-  /usr/sbin/fluent-gem list '^fluent-plugin-' --no-versions --no-verbose > %{local_base_plugins}
+  . %{_sysconfdir}/sysconfig/@SERVICE_NAME@
```

Comment:
I think this can fail when updating from v4.
For some reason, the current test is successful.
I will check these concerns together last.